assets/styles/scss/footer.scss

```diff
@@ -0,0 +1,80 @@
+@import "mixins.scss";
+@import "colors.scss";
+@import "style.scss";
+.fw-f-logo { 
+    position: relative;
+    right: 67px;
```

Comment:
it is better to use padding-right in parent container
try to avoid right with position relative
in most cases right/left/top/bottom is used only with fixed and absolute